.github/workflows/codeql-analysis.yml

```diff
@@ -2,41 +2,54 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ master ]
+    branches: [ main ]
   pull_request:
-    branches: [ master ]
-  schedule:
-    - cron: '15 21 * * 6'
```

Comment:
Did you remove the scheduled run on purpose?

Reply:
Yes, it's intentional. I don't see much value in running the check every Saturday. Do the queries and checks of CodeQL get updated that regularly?

Reply:
I guess it's not that critical for us, but it gets regularly updated with security issues and potential fixes.

Reply:
I share the opinion that is it's not (time-) critical for us. In the spirit of having dependencies only updated automatically once per month, I think running the checks only on PR and on main is sufficient.